e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -40,5 +36,29 @@ Scenario: Re-enrolling an agent
 @revoke-token
 Scenario: Revoking the enrollment token for an agent
   Given an agent is deployed to Fleet
+    And the agent is un-enrolled
+    And the "agent" process is "stopped" on the host
   When the enrollment token is revoked
-  Then an attempt to enroll a new agent fails
+  Then an attempt to enroll an agent with the old token fails
+
+@package-added-to-default-config
+Scenario: Adding packages to configurations
+  Given an agent is deployed to Fleet
+    And the package list service is available
+  When the "Cisco" "latest" package version is installed successfully
+    And the "Cisco" package datasource is added to the "default" configuration
+  Then the "default" configuration shows the "Cisco" datasource added
+
+@new-agent-configuration
```

Comment:
this is covered now in the agent-endpoint section, not needed to be repeated here when we resolve conflicts - we can just delete this scenario